Source/Applications/openXDA/openXDA/Controllers/WebAPI/WebAPI.cs

```diff
@@ -548,4 +549,37 @@ public class CustomerMeterController : XDAEndController<SystemCenter.Model.Custo
     public class ConsoleController : APIConsoleController {
         protected override IAPIConsoleHost Host => ServiceConnection.Default;
     }
+
+    [RoutePrefix("api/FTT")]
+    public class FTTController : ApiController
+    {
+        [HttpGet, Route("XDAEventList")]
+        public IHttpActionResult GetEventList([FromUri]string startdate, [FromUri] string enddate)
+        {
+            using (AdoDataConnection connection = new AdoDataConnection("systemSettings"))
+            {
+                string sqlQuery = @"SELECT     
+                    Event.ID as eventID,
+                    FaultSummary.Inception as event_datetime,
+                    Location.ShortName as substation,
+                    try_parse(Asset.AssetKey as int) as lineID,
+                    Asset.AssetName as line,
+                    CASE WHEN FaultSummary.Distance = '-1E308'
+                        THEN 'NaN'
+                    ELSE CAST(CAST(FaultSummary.Distance AS DECIMAL(16,2)) AS NVARCHAR(19))
+                    END AS distance_miles
```

Comment:
As I understand it, the original fields in `fttview` were aliased as...

* EventID
* event_datetime
* IsSelectedAlgorithm
* MeterName
* ShortName
* substation
* line
* linename
* distance_miles

I assume the renames are okay (`line` -> `lineID`, `linename` -> `line`).

`IsSelectedAlgorithm` is self evident due to the `WHERE` clause so that's okay to leave out.

Is it okay to remove `MeterName` and `ShortName`?